subsys/usb/class/bluetooth.c

```diff
@@ -137,6 +145,34 @@ static struct usb_ep_cfg_data bluetooth_ep_data[] = {
 	},
 };
 
+#define H4_CMD 0x01
```

Comment:
Should we get a BT_H4 header file and move some defines in there, we redefine these all over now.
Maybe some functionality could be moved there as well? like h4_recv/send common parts?

Reply:
If there are more driver that would benefit perhaps we could do that, but for the initial driver I rather have something simpler so we don't have to start introducing too many public APIs until we settle on the details.

Reply:
Sure, no need to do in this PR.

Reply:
Yes, will be done later on a separate PR.

Reply:
This is unused after last push.